### ISSUE TITLE
error server  

### ISSUE THREAD

**Issue by cejodrake** [FrontendMasters/fullstack-graphql#4]
![error](https://user-images.githubusercontent.com/3629120/72044025-94577000-3267-11ea-8241-e88e688748f3.JPG)

I can't execute the server could you help me please

**Comment by nvdai2401** [comment]
You should use the code in solution branch to fix this error.

**Comment by davidsonsns** [comment]
Yeah! You definitely (as @nvdai2401 said) will need to change your branch to `solution` (run `git checkout solution`). 

Just to clarify it, it's happening because the `master` branch was used only as of the base to the workshop, so:
- it's missing type definitions (fixed here https://github.com/FrontendMasters/fullstack-graphql/blob/solution/api/src/schema.js)
- it's missing some Apollo Server required options (fixed here https://github.com/FrontendMasters/fullstack-graphql/blob/solution/api/src/server.js#L6)

**Comment by rodrigoreeis** [comment]
> sim! Você definitivamente (como @ nvdai2401 disse) precisará alterar sua ramificação para `solution`(executar `git checkout solution`).
> 
> Apenas para esclarecer, está acontecendo porque o `master`ramo foi usado apenas a partir da base da oficina, portanto:
> 
> * falta definições de tipo (corrigidas aqui https://github.com/FrontendMasters/fullstack-graphql/blob/solution/api/src/schema.js )
> * faltam algumas opções obrigatórias do Apollo Server (corrigidas aqui https://github.com/FrontendMasters/fullstack-graphql/blob/solution/api/src/server.js#L6 )

Need add this code in client branch....

**Comment by rodrigoreeis** [comment]
> Yeah! You definitely (as @nvdai2401 said) will need to change your branch to `solution` (run `git checkout solution`).
> 
> Just to clarify it, it's happening because the `master` branch was used only as of the base to the workshop, so:
> 
> * it's missing type definitions (fixed here https://github.com/FrontendMasters/fullstack-graphql/blob/solution/api/src/schema.js)
> * it's missing some Apollo Server required options (fixed here https://github.com/FrontendMasters/fullstack-graphql/blob/solution/api/src/server.js#L6)

Need add this code in client branch...

**Comment by loiclaudet** [comment]
If you don't want to start with all solutions, just use `client` branch.

**Comment by kaboumk** [comment]
> If you don't want to start with all solutions, just use `client` branch.

Hello, j'ai fait ça mais la fenêtre de chrome reste tristement blanche et vide!
